packages/common/validators/src/validators/is-age.ts

```diff
@@ -1,17 +1,3 @@
-/**
- * @name isAge
- * @description
- * 주어진 문자열이 유효한 나이인지 검사합니다.
- *
- * ```typescript
- * function isAge(ageInput: string): boolean;
- * ```
- *
- * @example
- * isAge('25') // true
- * isAge('asd') // false
- * isAge('24.3') // false
- */
 export function isAge(ageInput: string): boolean {
```

Comment:
For now we have to add tossdocs-ignore to each of the source file 😢 (After migrating all of our docs out of jsdoc, we will remove all tossdocs-ignore

Reply:
yes! thank you
[fix: write /** @tossdocs-ignore */ plan to remove after migrate](https://github.com/toss/slash/pull/56/commits/7f66ba1ffd3898347e9b14b0d263d5f80d99cde1)
[7f66ba1](https://github.com/toss/slash/pull/56/commits/7f66ba1ffd3898347e9b14b0d263d5f80d99cde1)